jans-cedarling/cedarling/src/authz/mod.rs

```diff
@@ -339,6 +344,41 @@ impl Authz {
     }
 }
 
+/// Constructs the authorization context by adding the built entities from the tokens
+fn build_context(
+    config: &AuthzConfig,
+    request_context: Value,
+    entities_data: &AuthorizeEntitiesData,
+    schema: &cedar_policy::Schema,
+    action: &cedar_policy::EntityUid,
+) -> Result<cedar_policy::Context, BuildContextError> {
+    let namespace = config.policy_store.namespace();
+    let action_name = action.id().escaped().to_string();
+    let action_schema = config.policy_store.schema.json.
+        find_action(&action_name, namespace)
+        .map_err(|e| BuildContextError::FindActionSchema(action_name.clone(), e))?
+        .ok_or(BuildContextError::MissingActionSchema(action_name))?;
+    
+    let mut id_mapping = HashMap::new();
+    for entity in entities_data.iter() {
+        // we strip the namespace from the type_name then make it lowercase
```

Comment:
why we need to make lowercase for type name?

Reply:
the type name from the uid would be something like `Jans::Id_token`. then the key in the context schema is lowercase. for example:
```cedar
Jans {
    action View appliesTo {
        principal: [User],
        resource: [File],
        context: {
            "status": String,
            "id_token": Id_token,  <-- lower-case key in the record
        },
    };
}
```

Reply:
the type name from the uid would be something like `Jans::Id_token`. then the key in the context schema is lowercase. for example:
```cedar
Jans {
    action View appliesTo {
        principal: [User],
        resource: [File],
        context: {
            "status": String,
            "id_token": Id_token,  <-- lower-case key in the record
        },
    };
}
```

Reply:
this probably wont work if the user wants to do something like:
```cs
context: {
    "status": String,
    "my_custom_named_token": Id_token,
},
```
but given that having tokens passed to the context is Cedarling-specific, maybe make the user abide with this convention?

Reply:
Will current solution work if token is not defined in context schema?

Reply:
Looks like work. Great

Reply:
> I got it, you get type name map it to lower case and use it as key in the map.
I thought that if in schema type of token is same as in the schema you will put token entity in this place...

We can probably support custom namespace in the future by having a better schema parser but it just doesn't support it right now and it's out of scope so I just though let's do it like this for now.